scripts/apps/authoring/views/article-edit.html

```diff
@@ -7,6 +7,10 @@
 
     <label class="field__label">{{label('headline')}}</label>
     <span ng-if="schema.headline.maxlength" sd-character-count data-item="item.headline" data-html="true" data-limit="schema.headline.maxlength"></span>
+    <sd-validate-characters ng-if="schema.headline.validate_characters"
+        class="disallowed-char-error"
```

Comment:
Can you define the class name inside the React component?

Reply:
I did that first, but it due to this style was breaking so removed that.
![validate_char](https://user-images.githubusercontent.com/10600741/69544412-ab8f1680-0fb5-11ea-8ab0-f79c08a1ab6e.png)

But this should look like this.
![valid_fromat](https://user-images.githubusercontent.com/10600741/69544546-f6a92980-0fb5-11ea-9f2c-651751e490c3.png)

Reply:
:+1: keep it as is then